cli.js

```diff
@@ -2,7 +2,7 @@
 // Simple CLI for KaTeX.
 // Reads TeX from stdin, outputs HTML to stdout.
 // To run this from the repository, you must first build KaTeX by running
-// `yarn install` and `yarn run build`.
+// `yarn install` and `yarn build`.
```

Comment:
I read that `yarn install` is the default behavior, so the first could be replaced by `yarn`.  Similarly elsewhere in this PR.

Reply:
@edemaine Done!